docs/09_firebase_deployment.md

```diff
@@ -93,13 +93,17 @@ Finally, your Firebase project configured and it's time to configure the Firebas
 
 ## Firebase SDK configuration
 
-To configure the Flutter for Web application to use recently created Firestore Database follow the next steps:
+By default, the Flutter already configured for a Web application by using the `auto-generated` Firebase configuration (So, after you deploy to Firebase, your app automatically pulls the Firebase configuration object from the Firebase project to which you've deployed).
 
+If you okay with the defaults, you can skip this configuration step.
+
+But if you want to configure a connection to the Firebase `manually`, follow the next steps:
 1. Open the [Firebase console](https://console.firebase.google.com/), choose your project
 and go to the project setting (tap on the setting gear icon near the `Project Overview` on top of the left panel and select `Project settings`.
 2. Scroll down and find your Firebase Web Application.
 3. Go to `Firebase SDK snippet` of your application, select `Config` and copy the generated code.
 4. Go to the `web/index.html` file in the application directory and replace the following piece of code with the copied one in step 3:
+
 ```
```

Comment:
It seems like currently, we should replace another piece of code since the default configuration changed. Also, the firebase provides only configuration JSON, but we should also call the `initializeApp` and `analytics` methods. It would be great to improve our guide to include this information. Also, I believe it will be great to specify a full example of the configuration user should get to simplify the configuration process.